domains/fusoras.json

```
@@ -0,0 +1,11 @@
+{
+    "description": "Porfolio website",
+    "repo": "https://github.com/FuSoraS/portfolio",
+    "owner": {
+        "username": "FuSoraS",
+        "email": "developer.FuSoraS@gmail.com"
+    },
+    "record": {
+        "CNAME": "FuSoraS.github.io"
```

`CNAME` record must be lowercased
```suggestion
        "CNAME": "fusoras.github.io"
```

---

Its ready